src/components/Index.vue

```diff
@@ -85,6 +89,12 @@ export default {
       this._localSearchStore.queryParameters = this.queryParameters;
     }
 
+    if (this.cache) {
+      this._localSearchStore.enableCache();
```

Comment:
This allows users to disable the cache without requiring to instantiate a search store manually.

At the beginning I didn't even think of having an `ais-index` component, but I guess it makes implementation really easy.